charts/argo-cd/values.yaml

```diff
@@ -2197,7 +2197,7 @@ notifications:
     annotations: {}
 
     # -- The name of the secret to use.
-    ## If not set and create is true, the default name 'argocd-notifications-secret' is used
+    ## If not set the default name 'argocd-notifications-secret' is used
```

Comment:
If we are removing `and create is true` in this comment, should it also be removed in the comment for `cm.name`?